data/json/items/gun/ups.json

```
@@ -68,7 +70,7 @@
     "modes": [ [ "DEFAULT", "3 rd.", 3 ], [ "BURST", "5 rd.", 5 ], [ "AUTO", "high auto", 15 ] ],
     "ammo_effects": [ "LASER", "PLASMA_BUBBLE", "INCENDIARY" ],
     "flags": [ "NO_UNLOAD", "NEVER_JAMS", "NO_UNWIELD", "NO_SALVAGE", "NO_REPAIR", "UNBREAKABLE_MELEE", "NON_FOULING", "USE_UPS" ],
-    "overheat_threshold": 200,
+    "overheat_threshold": 300,
```

```suggestion
    "overheat_threshold": 200,
```
The gun barely overheats as is. Requiring 33 successive shots before that happens. I dont think further increasing its overheat threshold is warranted.

---

This number was changed to apply the increase to the gun used when on the Mech, since it was spawned with the GUNMOD added.
I did wonder if the additional 17 shots would be too strong for me too,
I also thought it would be too strong, but the cooling_value is unchanged, so the wait time increases the more you use it,
I also thought that it would be better to make it as strong as possible because it is very difficult to meet with a mech and it is basically impossible to recharge it and add GUNMODs.

The guns that are removed from the Mechs are basically unusable unless they are mounted on a vehicle, so even if we add a GUNMOD to the Mechs, it would be very difficult to get them deep into the lab, and we thought that the players who would get the Mechs would already have no use for the town.